src/tasks/eth/035-U17-main-unichain/config.toml

```diff
@@ -0,0 +1,48 @@
+l2chains = [
+    {name = "Unichain", chainId = 130}
+]
+
+templateName = "OPCMUpgradeV500"
+
+allowOverwrite = ["SuperchainConfig"]
+
+
+
+# Unichain (Permissionless)
+
+[[opcmUpgrades]]
+chainId = 130
+absolutePrestate = "0x03799051d2bfe459127d4597f469f535ff1bd2a6e1e2134443167620871c11f3"  # prestates."1.8.0-rc.3" https://github.com/ethereum-optimism/superchain-registry/blob/c9881d543174ff00b8f3a9ad3f31bf4630b9743b/validation/standard/standard-prestates.toml#L6
+expectedValidationErrors = "PROXYA-10,DF-30,PDDG-DWETH-30,PDDG-VM-30,PLDG-DWETH-30,PLDG-VM-30"
+
+
+
+[addresses]
+
+OPCM = "0x9e86a129e86a570fb13e89bd5e9aa98b642ae4c5" # Source: https://www.notion.so/oplabs/op-contracts-release-for-U17-Jovian-294f153ee16280b99539c5b6de67c72f?source=copy_link#295f153ee16280fe9c9af13970a5fac6
+StandardValidatorV500 = "0xE0b673cB5Da8DaC181d4D712eA563c4B14230286" # Source: cast call validator's from OPCM above
+SuperchainConfig = "0x95703e0982140D16f8ebA6d158FccEde42f04a4C" # Source: https://github.com/ethereum-optimism/superchain-registry/blob/6748f8050bfab9930abdfc46288f9629af2f7b38/superchain/configs/mainnet/superchain.toml#L3C27-L3C69
+SuperchainConfigProxyAdmin = "0x543bA4AADBAb8f9025686Bd03993043599c6fB04" # Source: cast call ProxyAdmin address from SuperchainConfig above
```

Comment:
May require template changes, but
- We can remove `StandardValidatorV500` and read it from the OPCM via `opcmStandardValidator()`
- We should also be able to remove the SuperchainConfig and SuperchainConfigProxyAdmin and fetch those from the address registry in the template.

Reply:
Agreed with @Wazabie in Slack that we will leave this for now to avoid changing the template.